docs/ccoctl.md

```diff
@@ -215,3 +215,124 @@ This command will delete the service id from the IBM Cloud
+ram:AttachPolicyToUser
+ram:CreateUser
+ram:GetUser
+ram:DeleteUser
+ram:CreateAccessKey
+ram:ListAccessKeys
+ram:DeleteAccessKey
+   ```
+
+3. Use the selected RAM user’s accesskey id/secret to configure the Alibaba Cloud SDK client's credential provider chain with [Envionment Creadentials](https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/docs/2-Client-EN.md#1-environment-credentials) mode or through [Credentials File](https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/docs/2-Client-EN.md#2-credentials-file) mode
+
+### Procedure
+
+1. Extract the list of CredentialsRequest custom resources (CRs) from the OpenShift Container Platform release image:
+
+   ```bash
+   $ oc adm release extract --credentials-requests --cloud=alibabacloud --to=<path_to_directory_with_list_of_credentials_requests>/credrequests quay.io/<path_to>/ocp-release:<version>
+   
+   ```
+
+**step 2&3 are only needed when preparing for upgrading clusters with manually maintained credentials. When doing a fresh installation please skip step 2&3**
+
+2. For each CredentialsRequest CR in the release image, ensure that a namespace that matches the text in the spec.secretRef.namespace field exists in the cluster. This field is where the generated secrets that hold the credentials configuration are stored.
+
+   Sample Alibaba Cloud CredentialsRequest object
+
+   ```yaml
+   apiVersion: cloudcredential.openshift.io/v1
+   kind: CredentialsRequest
+   metadata:
+     name: cloud-credential-operator-ram-ro
+     namespace: openshift-cloud-credential-operator
+   spec:
+     providerSpec:
+       apiVersion: cloudcredential.openshift.io/v1
+       kind: AlibabaCloudProviderSpec
+       statementEntries:
+       - action:
+         - ecs:CopySnapshot
+         - ecs:DeleteDisk
+         - ecs:DescribeInstanceAttribute
+         - ecs:DescribeInstances
+         effect: Allow
+         resource: '*'
+     secretRef:
+       namespace: cloud-credential-operator-ram-ro-creds
+       name: openshift-cloud-credential-operator
+   ```
+
+ 3. For any `CredentialsRequest` CR for which the cluster does not already have a namespace with the name specified in `spec.secretRef.namespace`, create the namespace:
+
+   ```bash
+   $ oc create namespace <component_namespace>
+   ```
+
+4. Use the `ccoctl` tool to process all `CredentialsRequest` objects in the `credrequests` directory:
+
+   ```bash
+   $ ccoctl alibabacloud create-ram-users --name <name> --region=<region> --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests --output-dir=xxxxxx
+   ```
+
+    where:
+
+   - `name` is the name used to tag any cloud resources that are created for tracking. 
+   - `region` is the Alibaba Cloud region in which cloud resources will be created.
+   - `credentials-requests-dir` is the directory containing files of component CredentialsRequests.
+   - `output-dir`/manifests is the directory containing files of component credentials secret.
```

Comment:
What does `/manifests` on this line mean?

Reply:
This `/manifests` means that results from the `ccoctl` binary will be placed in a directory on the system located at `/manifests`.

Reply:
I'm interpreting that to mean `<output-dir>/manifests` then - let me know if that's incorrect

Reply:
@jeana-redhat your interpretation is correct, the manifests will be placed inside `<output-dir>/manifests`. Let me share some examples:
If I run:
``` bash
ccoctl alibabacloud create-ram-users \
  --region ${ALIBABA_REGION_ID} \
  --name <cluster_name> \
  --credentials-requests-dir ${PWD}/cco-credrequests \
  --output-dir ${PWD}/cco-manifests
```
I will get that structure:
``` bash
$ tree cco-manifests/
cco-manifests/
└── manifests
    ├── openshift-cluster-csi-drivers-alibaba-disk-credentials-credentials.yaml
    ├── openshift-image-registry-installer-cloud-credentials-credentials.yaml
    ├── openshift-ingress-operator-cloud-credentials-credentials.yaml
    └── openshift-machine-api-alibabacloud-credentials-credentials.yaml

```